tests/testsuite/yank.rs

```diff
@@ -0,0 +1,47 @@
+//! Tests for the `cargo yank` command.
+
+use std::fs::{self, File};
+use std::io::prelude::*;
+
+use cargo_test_support::project;
+use cargo_test_support::registry::{self, api_path, registry_url};
+
+fn setup(name: &str, version: &str) {
+    fs::create_dir_all(&api_path().join(format!("api/v1/crates/{}/{}", name, version))).unwrap();
```

Comment:
I think this can be simplified a little with:

```rust
    let dir = api_path().join(format!("api/v1/crates/{}/{}", name, version));
    dir.mkdir_p();
    fs::write(dir.join("yank"), r#"{"ok": true}"#).unwrap();
```

`mkdir_p` needs `use cargo_test_support::paths::CargoPathExt;`